lib/kickbox/client.rb

```diff
@@ -11,9 +12,19 @@ def initialize(auth = {}, options = {})
       @http_client = Kickbox::HttpClient::HttpClient.new(auth, options)
     end
 
-    # 
-    def kickbox()
```

Comment:
maybe want to kep `def kickbox` for compatibility and just add a deprecation warning?